### ISSUE TITLE
Refresh Data Source not Refreshing

### ISSUE THREAD

**Issue by erica-sigma** [sigmacomputing/intercom_issues#17]
> Not sure if this is a bug, but every time I add a new table in snowflake and grant permissions to sigma, I have to refresh many times on the 'sources' ui to be able to see it

**Link:** [Conversation with Yoko Li](https://app.intercom.com/a/apps/gu1e0q0t/respond/inbox/all/conversations/23071813634)

**Customer:** [Yoko Li](https://app.intercom.com/a/apps/gu1e0q0t/users/5d1148a1142fedf31d1646df/all-conversations) from [Sutter Hill Ventures](https://app.intercom.com/a/apps/gu1e0q0t/companies/5d0bd3b04ade301d7d1401b9)
**Customer ID:** u4wIj3z0mtXPJN6swAAghfExtpnx6
**Company ID:** a9a8efce-9a82-487c-9e59-578d98e033e3

___

Intercom Operator will send a note to [this conversation](https://app.intercom.com/a/apps/gu1e0q0t/respond/inbox/all/conversations/23071813634) when this issue is closed.

**Comment by donhcd** [comment]
yeah so I'm not sure what the "'sources' ui" is, so I'm not sure what they're doing

**Comment by erica-sigma** [comment]
I assume she was talking about the connections panel on the left rail, but she may have been talking about the table selector when creating a worksheet. Both have a refresh button. 

<img width="305" alt="Screen Shot 2019-08-02 at 3 32 30 PM" src="https://user-images.githubusercontent.com/36901929/62402050-ca672a00-b53a-11e9-882d-cab2493f9fc7.png">

<img width="1278" alt="Screen Shot 2019-08-02 at 3 33 30 PM" src="https://user-images.githubusercontent.com/36901929/62402090-f71b4180-b53a-11e9-9bc1-3ffd9d3c9242.png">
